src/Microsoft.ApiDesignGuidelines.Analyzers/Microsoft.ApiDesignGuidelines.Analyzers.md

```diff
@@ -170,6 +240,34 @@ Severity: Warning
 
 Help: [https://msdn.microsoft.com/en-us/library/bb264484.aspx](https://msdn.microsoft.com/en-us/library/bb264484.aspx)
 
+### CA1065: Do not raise exceptions in unexpected locations ###
+
+A method that is not expected to throw exceptions throws an exception.
+
+Category: Design
+
+Severity: Warning
+
+Help: [https://msdn.microsoft.com/en-us/library/bb386039.aspx](https://msdn.microsoft.com/en-us/library/bb386039.aspx)
+
+### CA1066: Type {0} should implement IEquatable<T> because it overrides Equals ###
```

Comment:
This is the only rule title with a substitution parameter. This looks more like the diagnostic message. I would have expected the title to be something like "Implement Equatable<T> when overriding Equals".

Also, these appear to be placeholders: they have titles but not yet descriptions. Did you intend to have the descriptions in this PR?